### PR TITLE
Updates for Cpp lessons for WS21/22

### PR DESCRIPTION
**Pull request by malwaru**

### CODE REVIEW

File: content/programming/cpp/IntermediateCpp/src/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (alex-mitrevski) [inline content/programming/cpp/IntermediateCpp/src/.vscode/settings.json:1]: Please remove this file

File: content/programming/cpp/IntermediateCpp/src/classes/Person.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
+
+#include "Person.h"
+
+
+// using namespace std;
```
Comment (alex-mitrevski) [inline content/programming/cpp/IntermediateCpp/src/classes/Person.cpp:5]: Why is this left commented out?

File: content/programming/cpp/IntermediateCpp/src/classes/classes.cpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    Person p2("Tom","Hanks");
+    std::cout << p2.GetFirstName() << std::endl;
+
+
+    Person p3("Charlie","Chaplin",101);
+    std::cout << p3.GetFirstName() << std::endl;
+    std::cout << p3.GetAge() << std::endl;
+    
+
```
Comment (alex-mitrevski) [inline content/programming/cpp/IntermediateCpp/src/classes/classes.cpp:39]: Why so many empty lines here?

File: content/programming/cpp/IntermediateCpp/src/pointers/pointers.cpp [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    cout<<"\n Pass by value after "<<pass_val<<endl;
+    cout<<"Pass by reference after "<<pass_ref<<endl;
+
+
+    cout<<"\n\nPass by pointer before "<<*pAge<<endl;
+    func_point(pAge);
+    cout<<"Pass by pointer after "<<*pAge<<endl;
+
+
```
Comment (alex-mitrevski) [inline content/programming/cpp/IntermediateCpp/src/pointers/pointers.cpp:78]: Why so many empty lines?

File: content/programming/cpp/IntroToCpp/src/helloWorld/calulator.cpp [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        cout<<mul(num1,num2)<<endl;
+    }
+
+
+    if(action=="divi")
+    {
+        cout<<divi(num1,num2)<<endl;
+    }
+
```
Comment (alex-mitrevski) [inline content/programming/cpp/IntroToCpp/src/helloWorld/calulator.cpp:72]: Why so many empty lines?

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]
In addition to the other comments: What are the included binary files without extensions?

**Comment by malwaru** [comment]
> In addition to the other comments: What are the included binary files without extensions?

I think this merge is not required Alex. I added extra material to explain during the lecture. I shared those filed on Slack . So please ignore the pull request

**Pull request closed without merging**